joomla/joomla-cms
# [J4.0] Better "Your Modules" Article

Pull Request for Issue #36851 .

### Summary of Changes
Better "Your Modules" article making the article consistent with J4


### Testing Instructions
Install sample data and check the Your Modules article and the modules in the back-end


### Actual result BEFORE applying this Pull Request
---


### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/906604/151252969-8342bd05-de2f-42cc-b43e-d70627eaba59.png)



### Documentation Changes Required
Language change
For the translator check the name of the modules in your language
